src/pwa/components/Contexts/Context.js

```diff
@@ -89,11 +90,14 @@ class Context extends Component {
     } = this.props;
     return (
       <Fragment>
-        {bar === 'single' && <PostBar key="post-bar" />}
-        <React.unstable_ConcurrentMode>
-          {bar === 'list' && <ListBar key="list-bar" />}
-          {bar === 'media' && <MediaBar key="media-bar" />}
-        </React.unstable_ConcurrentMode>
+        <NavbarWrapper>
+          <SlotBeforeNavbar />
```

Comment:
This is only a question: why do we need two `before navbar` slot injectors? Is the one in `Theme` not enough?